routing/features_road_graph.cpp

```diff
@@ -253,6 +253,39 @@ double FeaturesRoadGraph::GetSpeedKMPHFromFt(FeatureType const & ft) const
   return m_vehicleModel.GetSpeed(ft);
 }
 
+void FeaturesRoadGraph::ExtractRoadInfo(FeatureID const & featureId, FeatureType const & ft,
+                                        double speedKMPH, RoadInfo & ri) const
+{
+  ri.m_bidirectional = !IsOneWay(ft);
+  ri.m_speedKMPH = speedKMPH;
+
+  ft.ParseGeometry(FeatureType::BEST_GEOMETRY);
```

Comment:
Yes. Code for serailization/deserialization features will be moved to indexer/feature_altitude unit.